pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -263,7 +263,7 @@ public void onTerminate() {
         synchronized (isReady) {
             try {
                 logger.info("Starting up DSM");
-                new DSMConfig(cfg);
+                DSMConfig.setConfig(cfg);
```

Comment:
I was puzzled by why we had a number of these new-ing up of DSMConfig without assignment.  I realized what's really going on is setting the config and not newing up a new instance, so I changed the constructor to a setter, since that more accurately reflects what's going on.